extensions/rbac/rbac.go

```diff
@@ -0,0 +1,18 @@
+package rbac
+
+const (
+	RestrictedAdmin               = "restricted-admin"
+	StandardUser                  = "user"
+	RoleClusterOwner              = "cluster-owner"
+	RoleClusterMember             = "cluster-member"
+	RoleProjectOwner              = "project-owner"
+	RoleProjectMember             = "project-member"
+	RoleCreateNS                  = "create-ns"
+	RoleReadOnly                  = "read-only"
+	RoleCustomManageProjectMember = "projectroletemplatebindings-manage"
+	RoleCustomCreateNS            = "create-ns"
+	ActiveStatus                  = "active"
+	ForbiddenError                = "403 Forbidden"
+	KubeConfigTokenSettingID      = "kubeconfig-default-token-ttl-minutes"
+	DefaultNamespace              = "fleet-default"
+)
```

Comment:
This looks good! Same as my other comment, maybe something to think about - and probably out of scope for this PR. 

Just because these are mostly strings, you might find adding a stringer method beneficial as these expand. It can also help it separate these constants further by the custom types you create, f.e.:

```
type Role string

	ClusterOwner         Role     = "cluster-owner"
	ClusterMember       Role      = "cluster-member"
	ProjectOwner          Role   = "project-owner"
	ProjectMember        Role     = "project-member"
	CreateNS                  Role  = "create-ns"
	ReadOnly                  Role  = "read-only"
	CustomManageProjectMember  Role = "projectroletemplatebindings-manage"
	CustomCreateNS        Role    = "create-ns"

func (r Role) String() string {
	return string(r)
}
```

More examples can be found in `provisioninginput` package.

While writing the above snippet, I saw that `create-ns` already has two different variables. It can prevent that too.